generator/package.json

```diff
@@ -15,10 +15,11 @@
     "js-yaml": "3.13.1",
     "read-pkg": "3.0.0",
     "yargs": "12.0.5",
-    "webpack": "^4.0.0",
-    "html-webpack-plugin": "^3.2.0",
+    "webpack": "^5.36.2",
```

Comment:
I'm wondering why these are not devDependencies.

Reply:
For che-theia generator they are actually the production dependencies. As they used when we run the generator binary on the Theia sources.

Reply:
Hmh...isn't the webpack config executed as part of running webpack and therefore the dependencies would be satisfied from whatever project webpack is used in? So shouldn't the "assembly" bring the webpack plugin as devDependencies?

Reply:
Probably, it's a good idea to experiment with in one of the H'n'H days. :slightly_smiling_face: